### DISCUSSION TITLE
Unusual Juggluco screen layout?

### DISCUSSION THREAD

**Discussion by MartijnZ** [j-kaltes/Juggluco#179] [Q&A] [answered]
Hi,

I'm trying to setup Juggluco as part of the AAPS. I downloaded the ARM version from [https://www.juggluco.nl/Juggluco/download.html](url), and choose the 'drive' option. After installing the app on my phone (Samsung galaxy S20 FE, SM--G780G/DS), the screen layout doesn't seem right:
![Screenshot_20240723_220435_Juggluco](https://github.com/user-attachments/assets/d568112d-ca75-4855-85b1-9b2aaaac520a)

Is this indeed an incorrect screen layout? If so, how to fix it? If not: how to use it ;) ?

Cheers,

Martijn

**Comment by j-kaltes** [comment, marked as answer]
I see nothing wrong. The first time you started Juggluco, an introduction was displayed: https://www.juggluco.nl/Jugglucohelp/introhelp.html

**Reply by MartijnZ** [reply]
Thanks for the Swift reply. My bad... I was just confused by the screen looks (forced landscape, full-screen, no buttons) that I assumed something was wrong.

My apologies!

Cheers,

Martijn
